sdk/metric/aggregator/histogram/benchmark_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package histogram_test
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+
+	"go.opentelemetry.io/otel/api/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/histogram"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/test"
+)
+
+const inputRange = 1e6
+
+func benchmarkHistogramSearchFloat64(b *testing.B, size int) {
+	boundaries := make([]float64, size)
+
+	for i := range boundaries {
+		boundaries[i] = rand.Float64() * inputRange
+	}
+
+	values := make([]float64, b.N)
```

Comment:
Are these supposed to be used below in the benchmark for loop?

Reply:
Yes. :smile: Fixed.